dev/tools/gen_defaults/generated/used_tokens.csv

```diff
@@ -856,6 +856,31 @@ md.sys.elevation.level2,
 md.sys.elevation.level3,
 md.sys.elevation.level4,
 md.sys.elevation.level5,
+md.sys.motion.duration.extra-long1Ms,
+md.sys.motion.duration.extra-long2Ms,
+md.sys.motion.duration.extra-long3Ms,
+md.sys.motion.duration.extra-long4Ms,
+md.sys.motion.duration.long1Ms,
+md.sys.motion.duration.long2Ms,
+md.sys.motion.duration.long3Ms,
+md.sys.motion.duration.long4Ms,
+md.sys.motion.duration.medium1Ms,
+md.sys.motion.duration.medium2Ms,
+md.sys.motion.duration.medium3Ms,
+md.sys.motion.duration.medium4Ms,
+md.sys.motion.duration.short1Ms,
+md.sys.motion.duration.short2Ms,
+md.sys.motion.duration.short3Ms,
+md.sys.motion.duration.short4Ms,
+md.sys.motion.easing.emphasized.accelerate,
+md.sys.motion.easing.emphasized.decelerate,
+md.sys.motion.easing.legacy,
+md.sys.motion.easing.legacy.accelerate,
+md.sys.motion.easing.legacy.decelerate,
```

Comment:
It looks like this doesn't include the token `md.sys.motion.easing.emphasized`, corresponding to what the spec text calls "Emphasized easing". Is that intended?

Reply:
I believe it was added in a later tokens version (we're on 0.162)